plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version3/class-wc-rest-products-controller-tests.php

```diff
@@ -868,7 +869,7 @@ public function test_products_filter_exclude_status_with_include_any() {
 		$request->set_query_params(
 			array(
 				'include_status' => array( 'any' ),
-				'exclude_status' => array( 'private', 'draft' ),
+				'exclude_status' => array( ProductStatus::PENDING, ProductStatus::DRAFT ),
```

Comment:
It looks like a wrong constant is being used in this line (`pending` instead of `private`).